model/accounts_mgmt/v1/label_type.model

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright (c) 2019 Red Hat, Inc.
+Copyright (c) 2020 Red Hat, Inc.
```

Comment:
No need to update copyright year on existing files (unless the 2019 was a mistake and the file was actually created in 2020).

Reply:
This file was created couple of weeks ago, so 2020

Reply:
Thanks for fixing it then :)